### ISSUE TITLE
How to improve validation/test accuracy of fasttext modal?

### ISSUE THREAD

**Issue by John0801-online** [facebookresearch/fastText#1229]
Hello,
First of all, thank you for such an amazing library. I am using this library to predict the sentiments of the sentence by categorizing training data into five different labels. My training data contains 30K rows for each label. I tried using different hyperparameters and different preprocessing steps but still no luck :( **My Training accuracy is around 99% but Testing is only 60%. Is there any way I can improve that?** 

**Note: I tried different overfitting steps as well (changing of the epoch, learning rate, word-n-grams etc.) but no change** 

If anyone had the same issue and know how to make modal better. Please advise on this. How should I improve accuracy?  Any leads on this are appreciated :)

Thanks in advance.
